<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleClass</class>
 <widget class="QWidget" name="BattleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battle</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="ownSideThird">
      <item>
       <property name="text">
        <string>五星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三星小乔</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ownSideSecond">
      <item>
       <property name="text">
        <string>五星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三星小乔</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ownSideFirst">
      <item>
       <property name="text">
        <string>五星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三星小乔</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="otherSideFirst">
      <item>
       <property name="text">
        <string>五星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三星小乔</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="otherSideSecond">
      <item>
       <property name="text">
        <string>五星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三星小乔</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="otherSideThird">
      <item>
       <property name="text">
        <string>五星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四星曹操</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三星小乔</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="ownThirdPicture">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ownSecondPicture">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ownFirstPicture">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="otherFirstPicture">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="otherSecondPicture">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="otherThirdPicture">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>29</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>己方</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>对方</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="VS">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="battle">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>battle</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="displayBattleReport">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="ownProgressBar_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="otherProgressBar_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="ownProgressBar_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="ownProgressBar_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="otherProgressBar_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="otherProgressBar_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="click">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>抽奖</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="battle.qrc"/>
 </resources>
 <connections/>
</ui>
